integrations/Matomo/util.js

```diff
@@ -11,14 +10,14 @@ import { NAME } from "./constants";
 const goalIdMapping = (event, goalListMap, message) => {
   let revenue;
   if (message.properties) revenue = message.properties.revenue;
-  each((val, key) => {
+  goalListMap.forEach((val, key) => {
```

Comment:
why we are changing this?

Reply:
`each` is not working as expected. May be due to Map(). So thought of doing with `forEach` and it is working well.

Reply:
got it